### PR TITLE
[CSM-289] Add updateWallet endpoint

### PR DESCRIPTION
**Pull request by ajnsit**

### CODE REVIEW

**Comment by akegalj** [comment]
@ajnsit base branch should be 0.5-rc , not master

File: daedalus/src/Daedalus/ClientApi.purs [hunk 1]
```diff
@@ -102,6 +102,24 @@ newWallet :: forall eff . EffFn5 (ajax :: AJAX, crypto :: Crypto.CRYPTO | eff) S
 newWallet = mkEffFn5 \wSetName wsAssurance wsUnit mnemonic spendingPassword -> fromAff <<< map encodeJson <<<
     either throwError (B.newWallet $ mkCPassPhrase spendingPassword) $ mkCWalletInit wSetName wsAssurance wsUnit mnemonic
 
+-- | Get meta information from given wallet
+-- Arguments: wallet object/identifier, type, currency, name, assurance, unit
+-- Returns json representation of wallets within given wallet id
+-- Example in nodejs:
+-- | ```js
+-- | > api.updateAccount('1gCC3J43QAZo3fZiUTuyfYyT8sydFJHdhPnFFmckXL7mV3f@2147483648','CWTPersonal','ADA','Initial wallet','CWANormal',0).then(console.log)
```
Comment (Martoon-00) [inline daedalus/src/Daedalus/ClientApi.purs]: Lalki

File: daedalus/src/Daedalus/ClientApi.purs [hunk 2]
```diff
@@ -102,6 +102,24 @@ newWallet :: forall eff . EffFn5 (ajax :: AJAX, crypto :: Crypto.CRYPTO | eff) S
 newWallet = mkEffFn5 \wSetName wsAssurance wsUnit mnemonic spendingPassword -> fromAff <<< map encodeJson <<<
     either throwError (B.newWallet $ mkCPassPhrase spendingPassword) $ mkCWalletInit wSetName wsAssurance wsUnit mnemonic
 
+-- | Get meta information from given wallet
```
Comment (Martoon-00) [inline daedalus/src/Daedalus/ClientApi.purs]: I suppose, whole doc to this method should be modified ))

Comment (akegalj) [inline daedalus/src/Daedalus/ClientApi.purs]: @Martoon-00 we will remove the docs - and long term we will depricate client-api as well as frontend won't use it any more.

Comment (akegalj) [inline daedalus/src/Daedalus/ClientApi.purs]: @Martoon-00 do you mean in cardanodocs or here ?

Comment (Martoon-00) [inline daedalus/src/Daedalus/ClientApi.purs]: Here. Nice then

Comment (ajnsit) [inline daedalus/src/Daedalus/ClientApi.purs]: Great! I've removed the "WIP" label.

**Pull request merged**